config/server/tce.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
server root might not be the most appropriate place (don't want to pollute it over time). Why not part of common config (create folder and rename config.js => index.js)